azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/bindings/CosmosDBBinding.java

```diff
@@ -88,17 +95,27 @@ public String getConnectionStringSetting() {
     }
 
     @JsonGetter
-    public boolean isCreateIfNotExists() {
+    public String isCreateIfNotExists() {
```

Comment:
Any reason why using String instead of Boolean?

Reply:
If bool values false, it will not appear in the generated function.json,  and some required attributes(E.g createIfNotExists) will miss and function will not work in azure.